<?php

namespace Casio\Exceptions;

use Exception;

class AlreadyExistsException extends Exception {

    public function __construct() {
        parent::__construct('ALREADY_EXISTS', 501);
    }
}
